actor-typed-tests/src/test/scala/org/apache/pekko/actor/typed/scaladsl/DispatcherSelectorSpec.scala

```diff
@@ -16,8 +16,10 @@ package org.apache.pekko.actor.typed.scaladsl
 import com.typesafe.config.Config
 import com.typesafe.config.ConfigFactory
 import org.scalatest.wordspec.AnyWordSpecLike
-
 import org.apache.pekko
+import org.apache.pekko.actor.typed.DispatcherSelector
```

Comment:
can you remove the org.apache from these new imports?

Reply:
Thanks, fixed